projects/packages/identity-crisis/src/class-identity-crisis.php

```diff
@@ -0,0 +1,1026 @@
+<?php
+/**
+ * Identity_Crisis package.
+ *
+ * @package  automattic/jetpack-identity-crisis
+ */
+
+namespace Automattic\Jetpack;
+
+use Automattic\Jetpack\Assets\Logo as Jetpack_Logo;
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+use Automattic\Jetpack\Constants as Constants;
+use Automattic\Jetpack\Status as Status;
+use Automattic\Jetpack\Sync\Functions;
+use Automattic\Jetpack\Tracking as Tracking;
+use Jetpack_Options;
+use Jetpack_Tracks_Client;
+use WP_Error;
+
+/**
+ * This class will handle everything involved with fixing an Identity Crisis.
+ *
+ * @since 4.4.0
+ */
+class Identity_Crisis {
```

Comment:
Jetpack::sync_idc_optin needs to be migrated into IDC package. 

We can leave the function there call the IDC method.